src/condor_ce_run

```diff
@@ -102,6 +108,12 @@ def generate_submit_contents(job_info):
     else:
         job_info['arguments'] = ''
 
+    try:
+        if ce.x509_user_proxy_path():
+            job_info['use_x509userproxy'] = True
+    except OSError:
+        job_info['use_x509userproxy'] = False
+
```

Comment:
What's `job_info['use_x509userproxy']` if `ce.x509_user_proxy_path()` is false but doesn't raise an error? Do we have to worry about a KeyError down the line?

Reply:
I don't think we have to worry but safety first!